05/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <meta
+        http-equiv="X-UA-Compatible"
+        content="ie=edge"
+    >
+    <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link
+        rel="stylesheet"
+        href="./css/reset.css"
+    >
+    <link
+        rel="stylesheet"
+        href="./css/global.css"
+    >
+    <link
+        rel="stylesheet"
+        href="./css/mobile.css"
+    >
+    <link
+        rel="stylesheet"
+        media="(min-width: 800px)"
+        href="./css/desktop.css"
+    >
+</head>
+
+<body>
+
+    <header class="header row">
+        <div class="container header__container">
+            <div class="header__menu">
+                <input
+                    type="checkbox"
+                    id="menu-switcher"
+                >
+                <label
+                    for="menu-switcher"
+                    class="hamburger"
+                ></label>
+                <nav class="cell header__nav"></nav>
+            </div>
+            <img class="cell header__logo" />
+        </div>
+    </header>
+
+    <section class="content content--small row">
+        <div class="content__container--small container">
+            <article class="content__column--small small cell"></article>
+            <article class="content__column--small bigger cell"></article>
```

Comment:
Zastanawiam się czy jeśli mamy modyfikator to czy potrzebujmy dodatkowej klasy chodzi mi tutaj o `small`.
Chciałbym zaznaczyć, że modyfikator zawsze występuje z elementem lub blokiem więc powinno być: `content__column content__column--small`. Teraz moim zdaniem można wykorzystać `content__column--small` oraz `content__column--big` do określenia wielkości - nie trzeba robić dodatkowego bloku.

Reply:
ok, dziękuję :) poprawiłam. Czasem się zagmatwam w tych nazwach ;)